src/lib/components/toggle/toggle.types.ts

```diff
@@ -9,10 +11,12 @@ export type LabelElementProps = Omit<
   LabelHTMLAttributes<HTMLInputElement>,
   'color'
 >;
+
+type MultipleLabels = { label?: string; supportingLabel?: string };
 export interface CheckedButtonProps extends CheckedButtonElementProps {
   name: string;
-  activation?: boolean;
   defaultValue: boolean;
-  size?: 'small' | 'medium';
-  labels?: string | [string, string];
+  labels?: MultipleLabels;
+  activeIcon?: FC<SVGProps>;
```

Comment:
Why active and inactive are props? dosent make sense sir. this icon can be added on the Element. because on figma we cant costumize this Element

Reply:
Marco asked me to add customizable icons, for both states, "off" and "on"

Reply:
Instead `FC<SVGProps>` to `ReactNode`